addons/etcd/scripts/switchover-with-candidate.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+
+set -ex
+
+leaderEndpoint=${KB_LEADER_POD_FQDN}:2379
+candidateEndpoint=${KB_SWITCHOVER_CANDIDATE_FQDN}:2379
```

Comment:
I 've looked at the code, both KB_LEADER_POD_FQDN and KB_SWITCHOVER_CANDIDATE_FQDN lacks cluster domain.

it's better to submitt an issue.

Reply:
maybe also HEADLESS_SVC_FQDN, issued (#7876)[https://github.com/apecloud/kubeblocks/issues/7876]

Reply:
could concatenate a ${CLUSTER_DOMAIN} temporarily as a workaround